fieldservice_sale/models/sale_order.py

```diff
@@ -0,0 +1,231 @@
+# Copyright (C) 2019 Brian McMaster
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    fsm_location_id = fields.Many2one(
+        "fsm.location",
+        string="Service Location",
+        help="SO Lines generating a FSM order will be for this location",
+        compute="_compute_fsm_location_id",
+        precompute=True,
+        store=True,
+        readonly=False,
+    )
+    fsm_order_ids = fields.Many2many(
+        "fsm.order",
+        compute="_compute_fsm_order_ids",
+        string="Field Service orders associated to this sale",
+    )
+    fsm_order_count = fields.Integer(
+        string="FSM Orders", compute="_compute_fsm_order_ids"
+    )
+
+    @api.depends("order_line")
+    def _compute_fsm_order_ids(self):
+        for sale in self:
+            fsm = self.env["fsm.order"].search(
+                [
+                    "|",
+                    ("sale_id", "=", sale.id),
```

Comment:
@EdgarRetes I think this line is causing the issue.  

I would try this and see if it still works and avoids the warning
```suggestion
                    ("sale_id", "in", sale.ids),
```

from https://github.com/odoo/odoo/commit/6b907f92076271f6aec5239bc0d6d9f1ab7986b1